### PR TITLE
feat: init audit database

### PR DESCRIPTION
**Pull request by datphamcode295**
#### What's this PR do?

- [x] Init audit database

### CODE REVIEW

File: migrations/schemas/20230109154551-init-audit.sql [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    blockchain_audit_id uuid             DEFAULT NULL,
+    cycle DECIMAL DEFAULT 0,
+    average_score DECIMAL,
+    status audit_status,
+    flag audit_flag,
+    quarter TEXT DEFAULT NULL,
+    action_item_high DECIMAL DEFAULT 0,
+    action_item_medium DECIMAL DEFAULT 0,
+    action_item_low DECIMAL DEFAULT 0,
+    sysc_at TIMESTAMP(6) DEFAULT NULL
```
Comment (thanhpn) [inline migrations/schemas/20230109154551-init-audit.sql:38]: it should be 'sync_at'

Comment (datphamcode295) [reply migrations/schemas/20230109154551-init-audit.sql]: updated

File: migrations/schemas/20230109154551-init-audit.sql [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+CREATE TABLE IF NOT EXISTS audits (
+    id               uuid PRIMARY KEY DEFAULT (uuid()),
+    deleted_at       TIMESTAMP(6),
+    created_at       TIMESTAMP(6)     DEFAULT (NOW()),
+    updated_at       TIMESTAMP(6)     DEFAULT (NOW()),
+    project_id       uuid NOT NULL,
+    notion_db_id     uuid NOT NULL,
+    auditor_id     uuid NOT NULL,
+    name TEXT,
```
Comment (thanhpn) [inline migrations/schemas/20230109154551-init-audit.sql:62]: should we add a field for Participants

File: migrations/schemas/20230109154551-init-audit.sql [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    action_item DECIMAL,
+    duration DECIMAL,
+    audited_at TIMESTAMP(6) DEFAULT NULL,
+    sync_at TIMESTAMP(6) DEFAULT NULL
+);
+
+ALTER TABLE audits
+    ADD CONSTRAINT audits_project_id_fkey FOREIGN KEY (project_id) REFERENCES projects (id);
+
+CREATE TYPE severity AS ENUM (
```
Comment (thanhpn) [inline migrations/schemas/20230109154551-init-audit.sql:76]: Change value of this enum
Enum severity {
  high
  medium
  low
}

File: migrations/schemas/20230109154551-init-audit.sql [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+);
+
+ALTER TABLE audit_action_items
+    ADD CONSTRAINT audit_action_items_audit_id_fkey FOREIGN KEY (audit_id) REFERENCES audits (id);
+ALTER TABLE audit_action_items
+    ADD CONSTRAINT audit_action_items_action_item_id_fkey FOREIGN KEY (action_item_id) REFERENCES action_items (id);
+
+CREATE TABLE IF NOT EXISTS action_item_snapshots (
+    id               uuid PRIMARY KEY DEFAULT (uuid()),
+    deleted_at       TIMESTAMP(6),
```
Comment (thanhpn) [inline migrations/schemas/20230109154551-init-audit.sql:149]: what is the purpose of deleted_at ?

File: migrations/schemas/20230109154551-init-audit.sql [hunk 5]
```diff
@@ -0,0 +1,191 @@
+
+-- +migrate Up
+
+CREATE TYPE audit_status AS ENUM (
```
[the remark this answers is not in the document]
Comment (datphamcode295) [reply migrations/schemas/20230109154551-init-audit.sql:4]: updated

**Pull request merged**